device/core/src/module_client.ts

```diff
@@ -210,50 +191,39 @@ export class ModuleClient extends EventEmitter {
    *
    * @throws {ReferenceError}         If the connStr parameter is falsy.
    *
+   * @returns {module:azure-iothub.Client}
```

Comment:
>* @returns {module:azure-iothub.Client} [](start = 3, length = 39)

To quote DT: "wrong!".  the @method and @description tags you put in aren't necessary.  If you need proof, I think you can  look at some of the provisioning doc at docs.microsoft.com and see how much detail it pulled out of the prototype.